content/posts/2021-in-numbers.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "2021 in Numbers"
+date: 2022-01-04
+---
+
+# 2021 for Packit in numbers
+
+Let's take a look on the last year through some numbers.
+We would like to should you some interesting statistics and charts
+that can describe the work of Packit during the year 2021.
+
+## Github Application
+
+By now, we have `169` installations of our GitHub application and the `41` is from the last year.
```

Comment:
I'd say the 'by now' implies present perfect so I'd use either:

- By now, we've had `169` ...
- As of now, we have `169` ...